src/components/Gizmo.tsx

```diff
@@ -1,6 +1,6 @@
 import { SceneInfra } from 'clientSideScene/sceneInfra'
 import { sceneInfra } from 'lib/singletons'
-import { MutableRefObject, useEffect, useRef } from 'react'
+import { MutableRefObject, useEffect, useMemo, useRef } from 'react'
```

Comment:
`useMemo` isn't used here right?

Reply:
Nope, I tried it out and then determined it added no use